CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Python: Added OBJECT ENCODING command ([#1471](https://github.com/aws/glide-for-redis/pull/1471))
 * Python: Added OBJECT FREQ command ([#1472](https://github.com/aws/glide-for-redis/pull/1472))
 * Python: Added OBJECT IDLETIME command ([#1474](https://github.com/aws/glide-for-redis/pull/1474))
+* Python: Added PFCOUNT command ([#1474](https://github.com/aws/glide-for-redis/pull/1474))
```

Comment:
o_O

Reply:
Whoops, I'll change the link when I create the PR upstream